test/integration/tasks_test.rb

```diff
@@ -25,12 +19,13 @@
   end
 
   it "produces the current schema from checked in migrations" do
-    tasks["db:schema:dump"].execute
+    # Loading all tasks here results in a circular import due to Sentry::Tasks. Only loading the necessary task.
```

Comment:
this seems like a bandaid for a deeper problem, loading all tasks should be possible ...